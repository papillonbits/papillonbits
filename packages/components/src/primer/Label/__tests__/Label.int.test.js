import { regular } from './Label.int.story'

describe('<Label />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
