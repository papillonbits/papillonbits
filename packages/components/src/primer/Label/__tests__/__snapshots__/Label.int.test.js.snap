// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> Render must match regular() 1`] = `
<div>
  <span
    class="IssueLabel bg-blue text-white"
    title="Label: default label"
  >
    default label
  </span>
</div>
`;
